### DISCUSSION TITLE
DISCUSSION: DataFusion Meetup in (Eastern?) Europe

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#10342] [Ideas]
After our first meetup, (see https://github.com/apache/datafusion/discussions/8152 / https://github.com/apache/datafusion/discussions/8522) I have heard there may be interest in other meetups in different geographies. 

I would personally love to support such meetups (will promote, try to attend, can help find sponsors, etc) as much as possible but I don't have the bandwidth to actually organize them.

Anyone who is interested in attending or helping to organize please leave a comment

Other meetup discussions:
https://github.com/apache/datafusion/discussions/10347

**Comment by alamb** [comment]
I think @gruuya  mentioned he might be interested in this type of event

**Comment by gruuya** [comment]
I'm happy to say that I'll be organizing this meetup in Belgrade on 2024/09/27 (last Friday this September)! More details to follow soon.

UPDATE: More details about the event can be found in the dedicated discussion: https://github.com/apache/datafusion/discussions/11431
